### DISCUSSION TITLE
Search by not unique text field

### DISCUSSION THREAD

**Discussion by molotow11** [keystonejs/keystone#8187] [Questions] [answered]
Added field definition, like this:
`  
aliases: text({  
  label : 'Status aliases (comma separated)',  
}),  
`

I've expected that it will work with `where` searching, like this:
`where: {
    aliases: {
     contains: "test"
    }
  }
`

But, this field not added into UniqueInput GraphQL schema and I have the following error:
`Field "aliases" is not defined by type "StatusWhereUniqueInput"`

![11](https://user-images.githubusercontent.com/28062183/205566226-ddcf90ee-e35a-4f63-a515-5875b7a00e92.png)
![22](https://user-images.githubusercontent.com/28062183/205566595-d9fc235a-d0c7-4d9c-a672-e1b33568bb16.png)

$ node -v
v16.17.1
"@keystone-6/auth": "^5.0.1",
"@keystone-6/core": "^3.1.3",
 "@keystone-6/fields-document": "^5.0.2",

**Comment by borisno2** [comment, marked as answer]
Hi @molotow11 
Thanks for posting. It looks like you are calling the `findOne` query using the non-plural name on GraphQL (`status`) which means any field you are searching on needs to have `isIndexed` set to `'unique`.  If you want to use this `where` input you will need to use the plural, in your case `statuses`, which will return multiple statuses - you can see a list of example GrqphQL queries keystone generates based on a list called `user` here https://keystonejs.com/docs/graphql/overview.
